gulpfile.js

```diff
@@ -182,7 +181,8 @@ var buildStyles = function (done) {
 				removeAll: true
 			}
 		}))
-		.pipe(dest(paths.styles.output));
+		.pipe(dest(paths.styles.output))
+		.pipe(browserSync.stream());
```

Comment:
🔍 using `.stream()` allows the browser to render the compiled css without fully reloading